Source/WebCore/platform/mediastream/cocoa/DisplayCaptureSourceCocoa.cpp

```diff
@@ -67,12 +66,13 @@ CaptureSourceOrError DisplayCaptureSourceCocoa::create(const CaptureDevice& devi
     case CaptureDevice::DeviceType::Screen:
 #if HAVE(REPLAYKIT)
         return create(ReplayKitCaptureSource::create(device.persistentId()), device, WTFMove(hashSalts), constraints, pageIdentifier);
-#else
-#if HAVE(SCREEN_CAPTURE_KIT)
+#elif HAVE(SCREEN_CAPTURE_KIT)
         if (ScreenCaptureKitCaptureSource::isAvailable())
             return create(ScreenCaptureKitCaptureSource::create(device, constraints), device, WTFMove(hashSalts), constraints, pageIdentifier);
-#endif
-        return create(CGDisplayStreamScreenCaptureSource::create(device.persistentId()), device, WTFMove(hashSalts), constraints, pageIdentifier);
+        ASSERT_NOT_REACHED();
```

Comment:
Seems like this falls through to `case CaptureDevice::DeviceType::Window` in release builds?

Reply:
Indeed, please add a `return { }`